app/controllers/api/profile_controller.rb

```diff
@@ -54,7 +54,7 @@ def resource_serializer
     if current_user == restricted_user
       Restricted::UserSerializer
     else
-      CurrentUserSerializer
+      Full::UserSerializer
```

Comment:
Since we have the concept of `Full` serializers now (ie, serializers which include child records), makes sense to move the CurrentUserSerializer under that umbrella.